src/olympia/addons/tests/test_models.py

```diff
@@ -1067,7 +1067,7 @@ def test_newlines_inline(self):
         assert self.newlines_helper(before) == after
 
     def test_newlines_code_inline(self):
-        before = "Code tags aren't blocks.\n\n" '<code>alert(test);</code>\n\nSee?'
+        before = "Code tags aren't blocks.\n\n<code>alert(test);</code>\n\nSee?"
```

Comment:
Why does this test need changing?

Reply:
Just my `ruff` formatting in my editor fixing this string to be a single string